python/mlx/nn/layers/activations.py

```diff
@@ -140,6 +172,11 @@ def gelu_fast_approx(x):
 
 @_make_activation_module
 class Sigmoid(Module):
+    r"""Applies the element-wise function:
```

Comment:
Keep the first sentence of the doc string standalone for the sake of autosummary table:

e.g. `Applies the sigmoid function, element-wise.`